data_safe_haven/backend/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .backend import Backend
+from data_safe_haven.backend.backend import Backend
```

Comment:
Again, I find this import style overly verbose, but if you prefer it that's fine.

Reply:
Fixed in 741290b0bbc51de7d7222022485b16ee35b5566e